docs/docusaurus/docs/guides/connecting_to_your_data/fluent/batch_requests/how_to_request_data_from_a_data_asset.md

```diff
@@ -59,6 +59,31 @@ For `dataframe` Data Assets, the `dataframe` is always specified as the argument
 ```python name="tests/integration/docusaurus/connecting_to_your_data/fluent_datasources/get_existing_data_asset_from_existing_datasource_pandas_filesystem_example.py build_batch_request_with_dataframe"
 ```
 
+## Extract a Batch from a Batch Request (Optional)
+
+You can use the Python slice function to remove a subset of data from a Batch Request and use a specific selection of records to build Metrics, Validations, and Profiles.
+
+1. Run the following code to retrieve an entire table of data from a SQL datasource:
+
+    ```python
+    table_asset = datasource.add_table_asset(name=asset_name, table_name=asset_table_name)
+    ```
+2. Run the following code to define the column to slice:
+
+    ```python
+    table_asset.add_splitter_column_value('my_column_name')
+    ```
+3. Run the following code to slice and filter the column:
+
+    ```python
+    batch_request = table_asset.build_batch_request(options = {'filter_options'})
+
+4. Run the following code to return the batch:
```

Comment:
@kwcanuck so this wouldn't be returning the batch, but printing out the amount of different batches created. So it will return like "1" or "3" or something like that. 

if you just do `table_asset.get_batch_list_from_batch_request(batch_request = batch_request)` this will return the actual list of batches.

Reply:
@HaebichanGX I have updated the code sample.